index.html

```diff
@@ -14,6 +18,103 @@
+        </address>
+    </header>
+
+    <div>
+      <section>
+        <p>Customer-oriented, young and enthusiastic server with excellent interpersonal and communication skills.
+          Licensed in preparation and sale of alcoholic beverages, bar procedures and liquor laws.</p>
+      </section>
+    </div>
+    <div class="mid">
+      <div class="skills">
+        <h2>Skills</h2>
+        <div class="uls">
+          <ul>
+            <li>Organized and efficient</li>
+            <li>Quick problem solver </li>
+          </ul>
+          <ul>
+            <li>Committed team player</li>
+            <li>High energy</li>
+          </ul>
+          <ul>
+
+            <li>Flexible schedule capability </li>
+            <li>Thrives in fast-paced environment</li>
+
+          </ul>
+        </div>
+
+        <div class="work-history">
+          <h2>Work History</h2>
+          <div class="heading">
+            <h3>Server/Waiter/Bartender</h3>
+
+          </div>
+          <div class="content">
+            <div class="heading">
+              <div class="name-address">
+                <h4>Omonia</h4>
+                <p class="direction">-33rd street and Broadway, Astoria</p>
+              </div>
+              <date>03/2016 - 09/2017</date>
+            </div>
+            <div>
+              <ul>
+                <li>Consistently provided professional, friendly and engaging service</li>
+                <li>Displayed enthusiasm and knowledge about the restaurant's menu and products. </li>
+                <li>Developed and maintained positive working relationships with others to reach business goals. </li>
+                <li>Assisted co-workers whenever possible</li>
+                <li>Delivered quality service by providing a warm and welcoming environment.</li>
+              </ul>
+            </div>
+            <div class="heading">
+              <div class="name-address">
+                <h4>Taverna Kyclades East Village</h4>
+                <p class="direction"> – 228 1st ave, 14th street, Manhattan </p>
+              </div>
+              <date> 04/2017 - Current </date>
+            </div>
+            <div>
+              <ul>
+                <li>Fine dining experience.</li>
+                <li>In charge of the front of the house, Customer feedback, and quality function of the staff. </li>
+                <li>Responsible for closing out register at the end of the day, and making sure money count was
+                  correct.</li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="bottom">
+            <div class="edu">
+              <h2>Education</h2>
+
+              <h4>High School Diploma -- Robert F. Wagner HS </h>
+            </div>
+            <div class="certificate">
+              <h2>Certifications</h2>
+              <h4>Certificate from New York Bartending School </h4>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
```suggestion
           </div>
          </div>
        </div>
      </div>
    </div>
  </div>
```
See how this turns into a little bit of div soup?